index.d.ts

```diff
@@ -0,0 +1,36 @@
+type Buffer = unknown
```

Comment:
The `Buffer` class should be imported from `bare-buffer`.

Reply:
I can see achieving that only by making `bare-buffer` a direct dependency since AFAIK it's not possible to install only the typings from a npm package.

Other runtimes have packages only for global types like [Bun's](https://github.com/oven-sh/bun/tree/main/packages/bun-types). And well, we need a place to add our core typings + built-ins modules.

If we create a "@types/bare" repo, I would be a dev-dependency here. What do you think @kasperisager?

Reply:
Add a peer and dev dependency on `bare-buffer` and then it's on the consumer to install.